test/integration/testFirefoxWebext.js

```diff
@@ -0,0 +1,94 @@
+import { Builder, By, until } from 'selenium-webdriver';
+
+import * as chai from 'chai';
+import * as firefox from 'selenium-webdriver/firefox';
+
+import ListPage from './page_objects/list_page';
+
+const assert = chai.assert;
+const expect = chai.expect;
+
+describe('The Firefox Notes web extension', function() {
+  const timeout = 10000;
+  const defaultNoteTitle = "Welcome to Firefox Notes!";
+  let addon_id;
+  let addon;
+  let options;
+  var driver;
+
+  this.timeout(20000);
+
+  before(async function() {
+    options = new firefox.Options();
+    options.setPreference('xpinstall.signatures.required', false);
+    options.setPreference('extensions.install.requireBuiltInCerts', false);
+    options.setPreference('extensions.webapi.testing', true);
+    options.setPreference('extensions.legacy.enabled', true);
+  });
+
+  beforeEach(async function() {
+    driver = await new Builder()
+      .forBrowser('firefox')
+      .setFirefoxOptions(options)
+      .build();
+    addon = await driver.installAddon('firefox_notes.xpi');
+    // Get addon numerical id
+    await driver.setContext('chrome');
+    addon_id = await driver.executeScript(
+      'var Cu = Components.utils;' +
+      'const {WebExtensionPolicy} = Cu.getGlobalForObject(Cu.import(' +
+      '"resource://gre/modules/Extension.jsm", this));' +
+      'return WebExtensionPolicy.getByID(arguments[0]).mozExtensionHostname;',
+      addon
+    );
+    await driver.setContext('content');
+    //load index html twice, as once seems to not load the app correctly
```

Comment:
This sounds like a bug somewhere

Reply:
I agree. I just wanted to see if it was the same on CI. I'll file something